app/scripts/modules/kubernetes/src/pipeline/stages/runJob/configureJob.controller.js

```diff
@@ -168,6 +168,11 @@ module.exports = angular
       this.stage.containers[index].lifecycle.preStop = handler;
     };
 
+    this.onTolerationChange = tolerations => {
+      this.stage.tolerations = tolerations;
+      $scope.$digest();
```

Comment:
I think it is nicer to use `$applyAsync` here, but who knows.

Reply:
👍